.github/workflows/plan-examples.yml

```diff
@@ -42,21 +50,34 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tf_working_dir: ${{ fromJson(needs.getExampleDirectories.outputs.directories) }}
+        directory: ${{ fromJson(needs.getExampleDirectories.outputs.directories) }}
 
     steps:
-      - name: checkout-merge
-        if: "contains(github.event_name, 'pull_request')"
-        uses: actions/checkout@v3
-        with:
-          ref: refs/pull/${{github.event.pull_request.number}}/merge
+      # - name: checkout-merge
+      #   if: "contains(github.event_name, 'pull_request')"
+      #   uses: actions/checkout@v3
+      #   with:
+      #     ref: refs/pull/${{github.event.pull_request.number}}/merge
```

Comment:
@Zvikan do you know why we need the two different routes for checking out? plans only run on PRs so the standard checkout should suffice, no?

Reply:
According to https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/572
"Explicit checkout PR code as pull_request_target changes checkout behavior to default to source main."

Reply:
ah ok thanks - I put that back as it was